koku/reporting/migrations/0115_populate_source_uuid.py

```diff
@@ -0,0 +1,56 @@
+# Generated by Django 2.2.13 on 2020-06-15 20:27
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("reporting", "0114_adding_source_uuid")]
+
+    operations = [
+        migrations.RunSQL(
+            sql="""
+UPDATE acct10001.reporting_awscostentrylineitem_daily_summary as aws
```

Comment:
`acct10001`

What about all the other accounts?

Reply:
Whoops. Great catch @maskarb, I totally forgot to take these out for when I was doing manual testing.